server.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+// not using import, using require instead
+// requires/imports express
+const express = require('express');
+const cors = require('cors');
+require('dotenv').config(); // lets you use env file
+const weatherData = require('./data/weather.json');
+
+const app = express(); // gives back return value
+console.log(app);
+
+app.use(cors()); // acts as middleware - intermediary between server and requests
+
+app.get('/weather', handleGetWeather);
+
+
+
+function handleGetWeather(request, response) {
+  // sends weather array and 200 (ok) .status(200) response
+  console.log(request.query);
```

Comment:
- In task card 2, for using find(), it may look something like this:
- We are using the .find() method to discover which city the lat, lon and searchQuery information belong to. We will need to look into the JSON data to do this.

`const city = weatherData.find(city => city.city_name === request.query.searchQuery);